src/pages/course-details-page/components/main-content/components/reviews-bar/components/reviews-list/components/review/components/review-main/index.tsx

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+
+import styles from "./styles.module.scss";
+
+type ReviewMainProperties = {
+	text: string;
+};
+
+const ReviewMain: React.FC<ReviewMainProperties> = ({ text }) => {
```

Comment:
Is this component too small to separate? There is no separate logic here

Reply:
Yes, it didn't have, but now it does have some separate logic - a button that shows/hides more text.